appveyor.yml

```diff
@@ -7,7 +7,7 @@ install:
 before_build:
 - nuget restore ILSpy.sln
 build_script:
-- msbuild ILSpy.sln /v:minimal /p:ResolveNuGetPackages=false "/logger:%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
+- msbuild ILSpy.sln /v:minimal /p:ResolveNuGetPackages=false /t:BeforeBuild,Build "/logger:%ProgramFiles%\AppVeyor\BuildAgent\Appveyor.MSBuildLogger.dll"
```

Comment:
The AssemblyInfo.cs files are not generated in my fresh local test repository. It seems VS does not execute the `BeforeBuild` tasks anymore.
I fixed this by using `<Target Name="PreBuild" BeforeTargets="PreBuildEvent">` as generated by VS (added a dummy task using the UI) instead of `<Target Name="BeforeBuild">`, however then UpdateAssemblyInfo.exe will be executed once for each target framework, which is not only unnecessary, but, due to parallel build of both target frameworks we were running into race conditions on the build server. (Seems to work on my (slow) machine.)

Reply:
Issue was relative ordering of the import of **Sdk.targets** and the definition of the BeforeBuild target. I reversed the order to fix the build.